### ISSUE TITLE
Tails on Qubes live-usb

### ISSUE THREAD

**Issue by andrewdavidwong** [QubesOS/qubes-issues#1969]
"How to Run Tails from a Qubes Live CD":
https://garlicgambit.wordpress.com/2016/04/22/how-to-run-tails-from-a-qubes-live-cd

At least one user (in addition to the author) has reported that this works:

* Should this be linked from the Qubes website?
* Should the content be added to the documentation? (Request author submission?)

Related: #1343 

CC: @adrelanos, @bnvk, @mfc

**Comment by mfc** [comment]
this seems like a duplicate of #1343. it would be great to loop the author into that thread.

also, they are basically hacking together Whonix in Qubes Live USB using Tails. it would be really, really nice if the folks trying to do this collaborated with the Whonix folks in order to create an agreed upon torified-workstation-in-Qubes strategy (whether live or not), and then Whonix folks can focus on not-live and Tails folks can focus on live.

>  They could help by releasing a new version or by providing instructions how to create a Qubes live cd. 

we definitely **should** provide documentation on how to create Qubes live cd, so that interested folks (this person, Tails folks) can continue to hack on this issue and maybe even manage it at some point.

**Comment by andrewdavidwong** [comment]
> this seems like a duplicate of #1343

Can you explain why? I'm not denying that it's a duplicate, but it's not immediately obvious to me that it is. #1343 seems to be about running Tails in an HVM, while this seems to be about running Tails in Qubes live-usb.

**Comment by mfc** [comment]
you're right, I guess it is more entirely dependent on #1343, rather than a dupe.

it is not clear to me that is worthwhile, vs trying to get Tails better integrated into Qubes (through Whonix) and then having all the "Tails-like" functionality in the Qubes Live USB to begin with.

I will create an email thread with the creator of the post and we can discuss more.

**Comment by unman** [comment]
I'm amazed #1343 is still open: it was only ever a display problem, for which I gave solution. It's not an issue now.

Running tails in qubes is straightforward whether in Live or installed: all the folk here are doing is documenting precise steps.

**Comment by SparkyMcSpark** [comment]
I can also confirm that this is working since the tails jessie release (2.0 I think). See also this thread on the tails-dev mail list: https://mailman.boum.org/pipermail/tails-dev/2015-December/009913.html
I still have some mouse problems (sometimes while scrolling down on a page suddenly it starts scrolling up, mouse pointer lag,  small and fixed window size of the vm). However, I don't know if I'm the only one experiencing this and if this is a qubes or tails issue. I guess it is the missing gui agent in tails (or in debian in general).

**Comment by mfc** [comment]
so just to be clear I think this is thread should be deprioritized vs sharing documentation on running Tails on an installed Qubes system #1343, since Qubes Live USB is out-of-date / not recommended.

**Comment by mfc** [comment]
and to be more clear, I think this documentation/guide is misdirected because as soon as we have an updated Qubes live image then we'd have whonix in it, which accomplishes all of this much more simply for the user. and all Tails-specific benefits (anti-forensics like RAM wiping on shutdown, etc) are not implemented in this method since they'd need to be implemented in Qubes on Tails workstation (see #2024 for attempted run-down of these functionalities).

So I recommend closing this ticket and moving conversation towards #2024 re: Tails-like functionality in Qubes, or towards #1343 documenting running Tails as an HVM in an installed version of Qubes (unless someone would prefer to use this ticket to document this).

**Comment by adrelanos** [comment]
> all Tails-specific benefits (anti-forensics like RAM wiping on shutdown, etc) are not implemented in this method since they'd need to be implemented in Qubes on Tails workstation (see #2024 for attempted run-down of these functionalities).

These can only be implemented at the dom0 / xen level. Impossible to implement at the VM level.

**Comment by mfc** [comment]
yep agreed, I'm going to close this for #2024.
